clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1062,44 +1057,49 @@ A #DR_ISA_REGDEPS #instr_t contains the following information:
   instr_get_src().
 - ISA mode: is always #DR_ISA_REGDEPS.  This information can be obtained using
   instr_get_isa_mode().
-- encoding bytes: an array of bytes containing the #DR_ISA_REGDEPS #instr_t
+- Encoding bytes: an array of bytes containing the #DR_ISA_REGDEPS #instr_t
   encoding.  Note that this information is present only for decoded instructions
   (i.e., #instr_t generated by decode() or decode_from_copy()).  This information
   can be obtained using instr_get_raw_bits().
-- length: the length of the encoded instruction in bytes.  Note that this
+- Length: the length of the encoded instruction in bytes.  Note that this
   information is present only for decoded instructions (i.e., #instr_t generated by
-  decode() or decode_from_copy()).  This information can be obtained by accessing
-  the #instr_t length field.
+  decode() or decode_from_copy()).  This information can be obtained using instr_length().
+  Be aware that in Google Workload Traces the instruction fetch size of a
+  #dynamorio::drmemtrace::memref_t# and the instr_length() of the corresponding fetched
+  instruction do not match! For convenience reasons we kept the instruction fetch size to
```

Comment:
Not just convenience: it's so analyses/simulations of front-end behavior have the right sizes.

Reply:
Good point, and much better argument. Done.